vyper/semantics/analysis/local.py

```diff
@@ -479,8 +415,8 @@ def visit_For(self, node):
 
                 try:
                     with NodeMetadata.enter_typechecker_speculation():
-                        for n in node.body:
-                            self.visit(n)
+                        for start in node.body:
```

Comment:
not sure `start` is a good name here - how about `stmt`

Reply:
> How about statement or simply node? Im not sure why I changed this

because they are all of type `vy_ast.Stmt`! re. terminology -- i think `stmt` is the standard abbreviation here (in the codebase, and also the terminology used in the python ast)